install-dev/upgrade/sql/1.7.5.0.sql

```diff
@@ -64,7 +64,7 @@ ALTER TABLE `PREFIX_order_detail`
   CHANGE `product_reference` `product_reference` varchar(64) DEFAULT NULL;
 
 ALTER TABLE `PREFIX_product`
-  CHANGE `reference` `supplier_reference` varchar(64) DEFAULT NULL,
+  CHANGE `reference` `reference` varchar(64) DEFAULT NULL,
```

Comment:
Do we really need to run this command 2 times? (1.7.5.0 & 1.7.5.1)

Reply:
The one in 1.7.5.1 is there so that people who already are in 1.7.5.0 have their database fixed.